tests/wasmfs/wasmfs_create.c

```diff
@@ -87,6 +87,17 @@ int main() {
   printf("File contents: %s", buf3);
   assert(errno == 0);
 
+  // Try to make a file with a name that is greater than WASMFS_NAME_MAX.
```

Comment:
This comments says "greater" and the last comment says "equal to," but the file names are the same. Should they both say "equal to"?